src/components/ListView/ListViewItem.js

```diff
@@ -103,7 +133,13 @@ ListViewItem.propTypes = {
   /** Contents for left section of ListViewItem (usually ListViewIcon) */
   leftContent: PropTypes.node,
   /** Checkbox form input component */
-  checkboxInput: PropTypes.node
+  checkboxInput: PropTypes.node,
+  /** Flag to use compound expansion contents */
+  compoundExpand: PropTypes.bool,
+  /** Flag to show compound expansion contents */
+  compoundExpanded: PropTypes.bool,
+  /** Function triggered when compound expandable content is closed */
+  onCloseCompoundExpand: PropTypes.func
 };
 ListViewItem.defaultProps = {
```

Comment:
would be nice to add defaultProp for `onExpandClose` with default to `noop` so we can simplify the syntax above

Reply:
also missed in previous review... inside `toggleExpanded(index) {` method, the `index` is not used.